src/patterns/ro-interaction-relations-src.yaml

```diff
@@ -22,8 +22,13 @@
 
 # new one proposed in https://github.com/oborel/obo-relations/pull/522
 # TODO: allow contributor to 'flow through' all named entities by default
-- id: RO:0018002
-  conforms_to: ObjectPropertyDefinedByInteractionProcess
+- conforms_to: ObjectPropertyDefinedByInteractionProcess
+  uses_rolified_property:
+    id: RO:0018001
```

Comment:
note I re-ordered this so the IDs are monotonically increasing

Reply:
great! We should probably document this. I think for our own sanity it makes sense for people to allocate in consecutive blocks of N, and even have an ID range dedicated to pattern based additions